src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -148,7 +148,11 @@ struct MPIDI_OFI_win_request;
 /* Stores per-rank information for RMA */
 typedef struct {
     int32_t disp_unit;
-#ifndef USE_OFI_MR_SCALABLE
+#if MPIDI_OFI_ENABLE_RUNTIME_CHECKS
+    /* For MR_BASIC mode we need to store an MR key and a base address of the target window */
+    uint64_t mr_key;
+    uintptr_t base;
+#elif !MPIDI_OFI_ENABLE_MR_SCALABLE
```

Comment:
The thing that would make this whole patch simpler is if we could just remove this check and always have the `mr_key` and `base` fields in this struct. That's going to add 16 bytes to the `MPIDI_OFI_win_targetinfo_t` structure all of the time, but at the moment, we're not using it a lot so it shouldn't actually make much difference I think.

Thoughts @hajimefu / @pkcoff / @raffenet ?